packages/design-system-docs/src/pages/startup/components.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Using components
+weight: 3
+---
+
+<h2>Using React components</h2>
```

Comment:
Use markdown where we can in this doc, for example, change heading to use the ### syntax